### PR TITLE
Update README to give full instructions for running

### PR DESCRIPTION
**Pull request by richb-hanover**
The README now has a more expansive instructions for running the server and client.
I'm not sure how to explain the green/blue colors of the latency plot.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,83 @@
...
-This produces an plot image like:
-<img src="media/plot.png">
 
+The client produces a `.png` file showing charts (below)
+as well as a `.crr` file with the raw data.
+The top plot shows:
+
+* Download traffic rates for five seconds
+* Upload traffic rates for five seconds
+* Bi-directional traffic rates for five seconds.
```
Comment (Zoxc) [inline README.md:52]: 5 seconds is just a default, so I might make these more generic.

**Comment by Zoxc** [comment]
Thanks. I may do some tweaks and take a matching Windows picture.

The blue latency plot is the time from the client to the server (one direction). The green latency plot is the time from the server to the client (one direction). The black latency plot is the total time from the client to the server and back from the server to the client (both directions).

File: README.md (hunk 1 above)
Comment (richb-hanover) [reply README.md:52]: Perhaps, "Crusader creates three bursts of traffic (by default, five seconds each) of download only, upload only, then bi-directional traffic. Each burst is separated by several seconds of idle time."

Feel free to modify the rest as you see fit. Thanks!

PS I have a few other comments/questions - should I submit them as an Issue?

**Comment by richb-hanover** [comment]
For clarity: Are you going to take the Windows screen shot and make further tweaks to the README? Thanks.

**Comment by richb-hanover** [comment]
Thanks!

**Pull request merged**